Tasks/Yourasov/Task1/DepositCalculator.py

```diff
@@ -0,0 +1,7 @@
+amount = float(input('Enter deposit amount: '))
+term = float(input('Enter deposit term, years: '))
+rate = float(input('Enter deposit interest rate in %: '))
```

Comment:
this task is related to money, it's unwise to use float for this calculations because it may result in additional errors, -1 point